content/participants/ttjc@Aadi-27/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Aditya Raj
+type: "participant"
+name: "Aditya Raj"
+github-link: "https://github.com/Aadi-27/"
+linkedin-link: "https://linkedin.com/in/exploreraadi/"
+looking-for: "job"
+description: "self-taught developer, skills: React | React Redux | Node.js | Express | Figma"
+---
+
+Hi! I am Aditya Raj, a first-year b.tech dropout self-learning web development. Let me walk you through my intro, projects and blogs 
+
+### introduction
+
+I have always been fascinated by the way web works and wanted to learn what goes under the hood but been stuck in the race of go-to-college then get a job. But as soon as I found out that I am not getting enough value after paying the college in bulks, I dropped out and started learning web development, making web apps and learn the PRACTICAL skills I always wanted to. A passionate learner ready to speak in public and accept challenges head on.  
+
+## why are you taking this challenge?
+
+I need a job so that I can quickly level up my skills by working with some highly creative people in the industry and provide value with my skillset.
+
+## something interesting about you?
+
+I am a shayar in my free time and have saved many of my friends relationships and played a hand in few of their breakups too.
+
+### help I need?
+
+It would be good if someone helps me with building backends using latest techs like Hashura or AWS amplify and working properly with databases.
+
+### how to contact me?
+
+- [Twitter] 
+(https://twitter.com/ExplorerAadi/)
+- [LinkedIn]
```

Comment:
Please correct the markdown format and remove the space so that hyperlink could be visible.

Reply:
I resolved it and thank you so much for your time.
Now please accept it.